_docs/setup/kubernetes/upgrading-istio.md

```diff
@@ -12,36 +12,47 @@ for the Kubernetes environment.
 
 ## Overview
 
-This guide describes how to upgrade an existing Istio deployment (including
-both control plane and sidecar proxy) to a new release of Istio. The upgrade
-process could involve new binaries as well as other changes like configuration
-and API schemas. The upgrade process may involve some service downtime.
+This guide describes how to upgrade an existing Istio deployment (including both control plane and sidecar proxy) to a new release of Istio. The upgrade process could involve new binaries as well as other changes like configuration and API schemas. The upgrade process may involve some service downtime.   To minimize the service downtime, please ensure your istio control plane components and your application are highly available with multiple replicas.
```

Comment:
`downtime.   To minimize ` 

Remove two spaces from above.

Reply:
ack